gravitee-apim-console-webui/src/management/api/general-info/api-general-info-migrate-to-v4-dialog/api-general-info-migrate-to-v4-dialog.component.scss

```diff
@@ -0,0 +1,122 @@
+@use 'sass:map';
+@use '@angular/material' as mat;
+@use '@gravitee/ui-particles-angular' as gio;
+
+$typography: map.get(gio.$mat-theme, typography);
+
+:host {
+  .title {
+    text-align: left;
+    margin-bottom: 16px;
```

Comment:
Why not keep the default values for everything related to dialog titles and card titles? It is often preferable to have the same values / UI throughout the application no ?